### PR TITLE
Added solution to CodeChef pattern printing Problem

### PR DESCRIPTION
**Pull request by Ayush-Malik**
I had added a solution to an interesting pattern printing problem of CodeChef
@Vishruth-S #204 I had made the suggested changes

### CODE REVIEW

File: CodeChef_problems/PPATTERN/PPATTERN.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    num = int(input()) # Taking the number of rows input
+    lis = []
+
+    # CORE LOGIC
+    # This for loop runs from (-ve num to +ve num)
+    for i in range(-num + 1 , num):
+        if num - abs(i) != num:
+            lis.append(num - abs(i))
+    lis.append(0)
+    print(lis)
```
Comment (Vishruth-S) [inline CodeChef_problems/PPATTERN/PPATTERN.py:12]: Remove `print(lis)` (line 12) since this will print unnecessary output

**Review by Vishruth-S — CHANGES_REQUESTED** [review]
Please remove line number 12 `print(lis)` in your solution

File: CodeChef_problems/PPATTERN/PPATTERN.py (hunk 1 above)
Comment (Ayush-Malik) [reply CodeChef_problems/PPATTERN/PPATTERN.py]: @Vishruth-S I have removed the extra print statement

**Review by Vishruth-S — APPROVED** [review]
Good job 👍

**Pull request merged**